readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Onoprijcha.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Onoprijcha.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="921" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/2cdfa1ec-9591-48ce-bd60-f584eeacc653">